packages/react/src/components/pagination/pagination.test.tsx

```diff
@@ -71,7 +71,61 @@ describe('Pagination', () => {
             const wrapper = shallow(<Pagination totalPages={3} numberOfResults={12345} />);
             const label = findByTestId(wrapper, 'resultsLabel');
 
-            expect(label.text()).toEqual('12345 results');
+            expect(label.text()).toEqual('Pagination - 1-4115 of 12345 results');
+        });
+
+        test('should display first page results label when number of results is even', () => {
+            const wrapper = shallow(
+                <Pagination totalPages={6} numberOfResults={30} activePage={1} />,
+            );
+            const label = findByTestId(wrapper, 'resultsLabel');
+
+            expect(label.text()).toEqual('Pagination - 1-5 of 30 results');
+        });
+
+        test('should display second page results label when number of results is uneven', () => {
+            const wrapper = shallow(
+                <Pagination totalPages={6} numberOfResults={30} activePage={2} />,
+            );
+            const label = findByTestId(wrapper, 'resultsLabel');
+
+            expect(label.text()).toEqual('Pagination - 5-10 of 30 results');
```

Comment:
Ca devrait plutot etre "Pagination - 6-10 of 30 results", le result 5 etant deja dans la page 1.

Reply:
bonne question et je sait pas si on veut commencer à compter à partir de 0 non plus 🤔 
en ce moment ça donne ça : 
```
1-5 of 30 results
5-10 of 30 results
10-15 of 30 results
15-20 of 30 results
20-25 of 30 results
25-30 of 30 results
```

Reply:
Imagine plutot que chaque Item a un numero et c'est ces numero qui sont affiche dans chaque page.

```
Page1: Item 1 a Item 5
Page2: Item 6 a Item 10
Page3: Item 11 a Item 15
```

Reply:
Je dirais comme Savut. La pagination est pour les humains, "le premier résultat" devrait être l'item 1.

Reply:
Oui c'est exactement pour les humains. Et les chiffres doit etre troujours inclusivement, le but c'est d'etre clair et precis pour le user.

Imagine les semaines dans un calendrier: 1-7, 8-14, 15-21